collab-database/src/views/layout.rs

```diff
@@ -64,6 +64,21 @@ impl TryFrom<i64> for DatabaseLayout {
   }
 }
 
+impl From<lib0Any> for DatabaseLayout {
+  fn from(value: lib0Any) -> Self {
+    if let lib0Any::BigInt(layout_ty) = value {
+      match layout_ty {
+        0 => DatabaseLayout::Grid,
+        1 => DatabaseLayout::Board,
+        2 => DatabaseLayout::Calendar,
+        _ => Self::default(),
+      }
+    } else {
+      Self::default()
+    }
+  }
+}
+
```

Comment:
Very similar to TryFrom<i64> above. How can I simplify this?

Reply:
If we can use the default layout, which is the grid, then we can remove the TryFrom